src/Servers/IIS/IIS/src/Core/IISHttpContext.IHttpRequestLifetimeFeature.cs

```diff
@@ -12,6 +11,9 @@ internal partial class IISHttpContext : IHttpRequestLifetimeFeature
     {
         private CancellationTokenSource _abortedCts;
         private CancellationToken? _manuallySetRequestAbortToken;
+        private object _abortLock = new object();
+        private bool _preventRequestAbortedCancellation;
+        protected bool _connectionAborted;
```

Comment:
Might as well call it _requestAborted hear since the whole type has a request lifetime.

Reply:
I had a rename that I thought went through...